include/ofi_net.h

```diff
@@ -92,6 +92,14 @@ static inline uint64_t ntohll(uint64_t x) { return x; }
 #endif
 #endif
 
+#ifdef MSG_ZEROCOPY
+#define OFI_ZEROCOPY MSG_ZEROCOPY
+#define OFI_ZEROCOPY_SIZE 9000 /* arbitrary based on documentation */
```

Comment:
we may want to make this configurable, if it doesn't cost us anything, since it is somewhat dependent on the system it's running on

Reply:
Yes, I have a note in my notebook to make this configurable, including disabling it.